src/main/java/core/basesyntax/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.bookstore.controller;
+
+import core.basesyntax.bookstore.dto.BookDto;
+import core.basesyntax.bookstore.dto.CreateBookRequestDto;
+import core.basesyntax.bookstore.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")

```

Comment:
I'd recommend you add "API" before your endpoints. It shows that your application is a REST
```suggestion
@RequestMapping(value = "/api/books")
```